library/dialpad/build.gradle

```diff
@@ -0,0 +1,20 @@
+apply plugin: 'com.android.library'
```

Comment:
a minimal build file for the module, technically it could apply the java plugin, for consistency it uses kotlin